### DISCUSSION TITLE
MAJOR Authentication Code Problem!

### DISCUSSION THREAD

**Discussion by lenaraskovska** [Pryaxis/TShock#1930] [General]
I'm having a major problem with this "authentication code". Initially, it's supposed to appear in the command console after you've set up your server, but it isn't there, and I need it to become super admin. Can someone please tell me what the hell I'm doing wrong? All that's showing up is the following attachment. I feel like I'm missing something.
![Annotation 2020-05-27 024552](https://user-images.githubusercontent.com/33382791/82968150-8e6aed00-9fc4-11ea-96c5-9cab67e55459.png)

**Comment by QuiCM** [comment]
Not sure I understand - the auth code is in the picture you have posted. Run `/setup 1725221` in game and you'll be good to go

**Comment by moisterrific** [comment]
You need to go in-game and use the `/setup <numbers>` command. Then follow the instructions it tells you on how to add an owner account. 

Owner is just below superadmin, you can temporarily enable superadmin for 10 minutes by typing `/su`

**Comment by lenaraskovska** [comment]
I see, I should've made that connection then, but is there no way to permanently enable super admin? @moisterrific I went through the steps I just didn't realise set up WAS the auth, did they change it?

**Comment by moisterrific** [comment]
you can enable permanent superadmin for a user, just type `/user group <your player name> superadmin` into the console

Edit: And yes, "auth" was renamed to "setup"

**Comment by lenaraskovska** [comment]
I'm incredibly relieved, I thought I was being gaslighted by TShock's console and was close to tearing my keyboard a new one. Before I set my user to super admin, I've read that some people's stuff disappears and the user gets given beginning items? Should I worry about this or not? @moisterrific

**Comment by hakusaro** [comment]
This issue has been transferred from the TSAPI repo to the TShock repo and converted from an issue into a discussion.
